pom.xml

```diff
@@ -24,7 +24,7 @@
 
   <artifactId>ratis</artifactId>
   <groupId>org.apache.ratis</groupId>
-  <version>1.1.0-SNAPSHOT</version>
+  <version>1.1.0-eb66796d-SNAPSHOT</version>
```

Comment:
Out of curiosity: why changing this line and all the following?

Reply:
Ups, it was just an accident.

It was changed locally  to upload a snapshot version of Ozone. (We use fixed SNAPSHOT version during the development to avoid unexpected changes). It shouldn't be part of this patch, just forgot to remove it.

Now, I reverted it. Thanks the reminder...